BizHawk.Emulation.Cores/Computers/AppleII/AppleII.IStatable.cs

```diff
@@ -88,16 +86,6 @@ private void InitSaveStates()
 
 		private readonly JsonSerializer _ser = new JsonSerializer();
 
-		public void SaveStateText(TextWriter writer)

```

Comment:
This one is even worse... Apple II _only_ has text states, and the binary states are actually text states stuffed in more or less.  Why remove this?

Reply:
The next thing on my list is to refactor apple II states to binary